### DISCUSSION TITLE
where does eza get default colors from?

### DISCUSSION THREAD

**Discussion by soloturn** [eza-community/eza#706] [Q&A, Howto, Help] [answered]
as other programs like fd, mcfly have difficulties with their color scheme, and eza gets it right,  i was wondering how eza gets its default colors?

**Comment by PThorpe92** [comment, marked as answer]
The colors themselves are "hardcoded", meaning e.g. config type files are yellow, dirs are blue, etc... just using ansi escape codes from [ansiterm](https://github.com/rustadopt/ansiterm-rs) that will obviously show up differently depending on the specific color scheme you have setup with your specific terminal that determines exactly which hue to display for each of the color options. :+1: 

The problem is I don't know how else you would do this, as (to my knowledge) that is essentially the only way to colored text in a terminal. I know LSD has a config file you can use to specify color preferences (that is something we are working on too, currently you have to use ENV_VAR's for eza to change the colors of certain things).

Hope this answered your question :)

**Reply by zaxioms0** [reply]
Is there a way to view the defaults in `$EZA_COLORS` format?

**Comment by rpdelaney** [comment]
`$LS_COLORS` and `$EZA_COLORS` is a fine way of changing this. It follows [dircolors](https://www.gnu.org/software/coreutils/manual/html_node/dircolors-invocation.html)!
